docs/docs/feature/keymaps.md

```diff
@@ -65,7 +63,7 @@ be used for that key position.
 
 In this case, the `A` is actually a define for the raw HID keycode, to make keymaps easier to read and write.
 
-For example of a binding that uses two parameters, you can see how "mod-tap" (`mt`) is bound:
+For an example of a binding that uses two parameters, you can see how "mod-tap" (`mt`) is bound:
```

Comment:
Can we make this link to the newly added mod tap docs?

Reply:
Resolved in brainwart/zmk@7c2d2c61e32d4bb0b4896862c57a36214aaa390b